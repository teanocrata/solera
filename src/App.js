import React, { Component } from "react";
import "./App.css";
import SNavbar from "./navbar/SNavbar";
import Page from "./page/Page";

class App extends Component {
  render() {
    return (
      <div>
        <SNavbar />
        <Page />
      </div>
    );
  }
}

export default App;
