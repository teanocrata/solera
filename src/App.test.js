import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import App from "./App";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
});

test("App renders correctly", () => {
  const app = renderer.create(<App />).toJSON();
  expect(app).toMatchSnapshot();
});
