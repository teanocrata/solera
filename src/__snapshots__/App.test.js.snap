// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div>
  <nav
    className="navbar navbar-inverse navbar-fixed-top"
  >
    <div
      className="container"
    >
      <div
        className="navbar-header"
      >
        <a
          className="navbar-brand"
          href="/"
        >
          <img
            alt="Logo"
            className="Nav-logo"
            src="solera_logo_80x80_green.png"
          />
        </a>
        <button
          className="navbar-toggle collapsed"
          onClick={[Function]}
          type="button"
        >
          <span
            className="sr-only"
          >
            Toggle navigation
          </span>
          <span
            className="icon-bar"
          />
          <span
            className="icon-bar"
          />
          <span
            className="icon-bar"
          />
        </button>
      </div>
      <div
        aria-expanded={null}
        className="navbar-collapse collapse"
      >
        <ul
          className="nav navbar-nav navbar-right"
          role={null}
        >
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="https://www.clasesdeequitacionmadrid.com/instalaciones-hipica"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              Instalaciones
            </a>
          </li>
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="https://www.clasesdeequitacionmadrid.com/clases-de-equitacion"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              Clases
            </a>
          </li>
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="https://www.clasesdeequitacionmadrid.com/pupilaje"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              Pupilaje
            </a>
          </li>
          <li
            className="dropdown"
            style={undefined}
          >
            <a
              aria-expanded={false}
              aria-haspopup={true}
              className="dropdown-toggle"
              href="#"
              id="basic-nav-dropdown"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              Actividades
               
              <span
                className="caret"
              />
            </a>
            <ul
              aria-labelledby="basic-nav-dropdown"
              className="dropdown-menu"
              role="menu"
            >
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="menuitem"
                  tabIndex="-1"
                  xlinkHref="https://www.clasesdeequitacionmadrid.com/cumpleanos-con-caballos"
                >
                  Cumpleaños
                </a>
              </li>
              <li
                className=""
                role="presentation"
                style={undefined}
              >
                <a
                  href="#"
                  onClick={[Function]}
                  onKeyDown={[Function]}
                  role="menuitem"
                  tabIndex="-1"
                  xlinkHref="https://www.clasesdeequitacionmadrid.com/cumpleanos-con-caballos"
                >
                  Campamentos
                </a>
              </li>
            </ul>
          </li>
          <li
            className=""
            role="presentation"
            style={undefined}
          >
            <a
              href="https://www.clasesdeequitacionmadrid.com/blog"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              Blog
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
  <div>
    <div
      className="text-center jumbotron"
      style={
        Object {
          "backgroundColor": "transparent",
          "height": undefined,
        }
      }
    >
      <div>
        <h1>
          Visitas guiadas
        </h1>
        <p>
          <button
            className="btn btn-default"
            disabled={false}
            type="button"
          >
            Ven a conocernos
          </button>
        </p>
      </div>
    </div>
    <br />
    <div
      className="container-fluid"
    >
      <div
        className="show-grid row"
        style={
          Object {
            "backgroundColor": "white",
          }
        }
      >
        <div
          className="slideanim col-md-4"
        >
          <img
            className="img-circle"
            src="pony_girls_square.jpg"
          />
          <h3>
            Da de comer a los ponys
          </h3>
          <p>
            Los caballos y los ponys son animales muy sociables, detectan nuestras emociones y son muy curiosos, además ¡les encantan las frutisinas!
          </p>
        </div>
        <div
          className="slideanim col-md-4"
        >
          <img
            className="img-circle"
            src="farmer_square.jpg"
          />
          <h3>
            Conoce nuestro día a día
          </h3>
          <p>
            Una escuela de equitación tiene muchas actividades, desde hacer las camas, la comida o salir a jugar ¡igual que en casa!
          </p>
        </div>
        <div
          className="slideanim col-md-4"
        >
          <img
            className="img-circle"
            src="horse_training_square.jpg"
          />
          <h3>
            Monta en uno de nuestros ponys
          </h3>
          <p>
            La actividad estrella de nuestra visita, descubre la sensación de conectar con un pony, verás como te conoce, te escucha, y disfrutaréis juntos.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
